tests/simulation/test_simulator.py

```diff
@@ -112,6 +114,8 @@ def test_many_miners_since_beginning(self):
         for node in nodes[1:]:
             self.assertTipsEqual(nodes[0], node)
 
+    # retry the test once if it fails, see: https://github.com/box/flaky
```

Comment:
Comment could be "retry the test _more times_ if it fails..." because with runs=3 it's more than once.

Reply:
I'll fix it. Thanks!